### ISSUE TITLE
help

### ISSUE THREAD

**Issue by b1naryth1ef** [lollipopman/gmail3#3]
i tried running this program but it failed

```
[15-12-04 07:53:09:939 PST] Debugging disabled
[15-12-04 07:53:09:940 PST] Retrieved scriptState: null
[15-12-04 07:53:09:940 PST] No stored scriptState, creating an empty one
[15-12-04 07:53:10:674 PST] Error: No item with the given ID could be found, or you do not have permission to access it.
```

thanks

**Comment by lollipopman** [comment]
It appears that the script only works with Google Apps for work, I will investigate why

**Comment by lollipopman** [comment]
@b1naryth1ef should be fixed by b7f4a73b0aa16ea17a4575f2c2ae81e72c2b6efd, please let me know

**Comment by lollipopman** [comment]
Please reopen if you experience the problem again
